forem/forem
# Add feature flag for Connect

## What type of PR is this? (check all applicable)


## Description

This PR adds a feature flag for Connect so it can be en- or disabled on a per-Forem basis. It will be disabled by default for new Forems.

## Related Tickets & Documents

n/a

## QA Instructions, Screenshots, Recordings

1. With the feature flag disabled make sure you **don't** see the following:
     1. The Connect link in the top navbar
     2. A button to chat with someone on their profile page if they have their inbox set to open
     3. Connect settings in `/settings/extensions`
     4. Any mention of Connect in mod emails
2. With the feature flag enabled you should see all of the above.

### UI accessibility concerns?

n/a

## Added/updated tests?


## [Forem core team only] How will this change be communicated?


## Pre-deployment tasks

* [ ] Add and enable the feature flag for DEV:
    ```
    FeatureFlag.enabled(:connect)
    ```
